readme.md

```diff
@@ -20,7 +20,7 @@ Includes a bunch of [general Markdown rules](config.js) and some [Awesome specif
 
 The CLI requires Node.js and Git.
 
-Type the command `npx awesome-lint` followed by the URL of the repo you want to check:
+In a git directory, type the command `npx awesome-lint` followed by the URL of the repo you want to check:
```

Comment:
This becomes moot with https://github.com/sindresorhus/awesome-lint/issues/180#issuecomment-1839399718